load_tests/load_test.py

```diff
@@ -394,7 +394,7 @@ def parse_json_template(template, dict):
     for key in dict:
             if(key[0] == '$'):
                 data = data.replace(key, dict[key])
-            elif(key == OUTPUT_PLUGIN):
+            else:
```

Comment:
from slack:

this one is safe
previously it would relpace the vars if the key was for the. current output plugin
now it just iterates overall of them






it does a find and replace




so if the string isn't in our template, it own't replace it
this old logic was unneeded and wrong